hugo/content/3_how-tos/34_networking-and-communication/347_enable-vpn.md

```diff
@@ -69,7 +69,7 @@ To enable or disable VPN for an existing project, follow these steps:
 
   1. Navigate to the right navigation pane and click **Devices**.
   2. Select the specific device for which you want to enable VPN.
-  3. On the device details page, click **Enabled** next to the VPN option. For more information, see [Enable VPN for an Online Device](/3_how-tos/32_device-management/321_onboarding-a-device/#enable-vpn-for-an-online-device)
+  3. On the device details page, click **Enable VPN**.  A pop-up will appear with the **Advertise Routes** field. To advertise your device on the subnets specified during project creation, mark the checkbox. For more information, see [Enable VPN for an Online Device](/3_how-tos/32_device-management/321_onboarding-a-device/#enable-vpn-for-an-online-device)
```

Comment:
@SanchithaSridhar Does the pop-up appear for all cases or only when the project has subnets?

Reply:
yes it appears only if the projects have subnets, have updated the content